apps/editing-toolkit/editing-toolkit-plugin/global-styles/src/font-pairings-panel.js

```diff
@@ -4,43 +4,48 @@
 import { ENTER } from '@wordpress/keycodes';
 import classnames from 'classnames';
 import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import NoSupport from './no-support';
-import loadFontPairingPreview from './font-pairings-panel-previews';
+import { getFontTitle } from '../../../../../client/landing/gutenboarding/constants';
```

Comment:
Is there a better way to import this?

Reply:
I know in the calypso code it can be imported with just 'client/' but I think that is to do with the lerna packages setup. @apeatling do you know if there is a better way to do this within the editing toolkit, or if it is going to cause any issues pulling in a file from outside of the plugin folder like this?

Reply:
I'm not sure this is going to work? For example when the plugin is built and running on Atomic these files are not present.

Reply:
Having said that I'm completely unsure about that.

Reply:
I'm not actually sure about this. @noahtallen Any ideas on this one?

Reply:
I'm unfortunately not familiar, but I do share the concern! I wonder if it makes sense to move `getFontTitle` into its own package which is then imported by both Gutenboarding and the toolkit plugin?

Reply:
To take a step back here -- `getFontTitle` fetches the display text for a given font. At the moment, _all_ it does is map `Playfair Display` to `Playfair`:

<img width="236" alt="Screen Shot 2020-11-02 at 1 04 02 PM" src="https://user-images.githubusercontent.com/63313398/97919028-f22fd980-1d0b-11eb-9ac3-e5b84fb8b0a6.png">

I did this in order to match the way fonts are displayed in Gutenboarding, but it's worth pointing out that historically, global styles has just used the full text "Playfair Display":
<img width="287" alt="Screen Shot 2020-11-02 at 1 01 45 PM" src="https://user-images.githubusercontent.com/63313398/97918831-9e24f500-1d0b-11eb-8f44-bba91273bc7c.png">

So perhaps the better question is, do we need to do this at all?

Reply:
If that's all, then I think we can remove this for now and unblock this PR. 👍